starter-code/src/App.css

```diff
@@ -0,0 +1,93 @@
+/* .App {
+  margin: 1rem;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  background-color: tomato;
+}
+
+.App img {
+  width: 300px;
+} */
+
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.App-container {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.top {
+  background-color: #1f2535;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5% 0;
+}
+
+.bottom {
+  background-color: white;
+  color: #1f2535;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5% 0;
+}
+
+.discover {
+  width: 90%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+nav {
+  display: flex;
+  width: 90%;
+  justify-content: space-between;
+}
+
+.nav-img {
```

Comment:
instead of creating this class you could have targeted the tag using `nav > img `